### DISCUSSION TITLE
cmp emoji not worked for me

### DISCUSSION THREAD

**Discussion by OriLiMu** [LazyVim/LazyVim#1831] [Q&A] [answered]
I just install lazyvim. I want to change add source to nvim-cmp. I check the link.  
https://www.lazyvim.org/configuration/recipes#add-a-nvim-cmp-source already.  
The code in this link I already tried but not work. 

Turn back to the example.lua file.
I can see that the emoji already changed in the example.lua file after installation.

```lua
  -- override nvim-cmp and add cmp-emoji
  {
    "hrsh7th/nvim-cmp",
    dependencies = { "hrsh7th/cmp-emoji" },
    ---@param opts cmp.ConfigSchema
    opts = function(_, opts)
      table.insert(opts.sources, { name = "emoji" })
    end,
  },

```
I think the emoji should already added. So I try to use cmp-emoji by typing `:smile` nothing happened no 
completion window pop out. Why is this? How to fix?

**Comment by dpetka2001** [comment, marked as answer]
The `example.lua` is just there as an example. It doesn't actually do anything. You should create a file in `/lua/plugins/cmp.lua` and put the following contents inside that file like
```lua
return {
  {
    "hrsh7th/nvim-cmp",
    dependencies = { "hrsh7th/cmp-emoji" },
    ---@param opts cmp.ConfigSchema
    opts = function(_, opts)
      table.insert(opts.sources, { name = "emoji" })
    end,
  },
}
```
Then restart LazyVim so that `cmp-emoji` will be installed.
